ui/selectors/selectors.js

```diff
@@ -1778,7 +1779,9 @@ export function getShowRecoveryPhraseReminder(state) {
  * @returns Number of unapproved transactions
  */
 export function getNumberOfAllUnapprovedTransactionsAndMessages(state) {
```

Comment:
I see that this selector is used for the `routes` prop `unapprovedTransactions`. This prop name was already confusing before now for a few reasons: it's just the count, not the transactions themselves, and it's not just transactions (it includes non-transaction confirmations).

But it's slightly more confusing now that we're including unapproved confirmations across all chains. Perhaps we should take this opportunity to make the prop name more clear? e.g. something like `totalUnapprovedConfirmationCount`

Reply:
Updated!